src/cluster/websocket/WebSocket-IXWebSocket.cc

```diff
@@ -138,8 +139,13 @@ std::unique_ptr<WebSocketServer> StartServer(std::unique_ptr<WebSocketEventDispa
                                                   remoteIp = std::move(remoteIp),
                                                   ixws = std::move(ixws)](const ix::WebSocketMessagePtr& msg) mutable {
             if ( msg->type == ix::WebSocketMessageType::Open ) {
+                // Pass all headers in upper-cased form to the dispatcher.
```

Comment:
We only need a single header value, so this seems like overkill.

What do you think about instead pulling out the value for `X-Application-Name` (since this now effectively an API for us it seems fine to require just that case), and store its value as `std::optional<std::string> app` in `WebSocketOpen`? The code setting up the backend could then compute a value `std::string app = open.app ? *open.app : "unknown";` (`TODO` re: "sanity" still applies, but IMO fine to punt on).

Reply:
Yeah, good with me, thanks!